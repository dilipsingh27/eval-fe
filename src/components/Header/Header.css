*{
    margin:0;
    padding:0;
}

.header {
    background-color: black;
    height:10vh;
}

.para {
    font-size: 40px;
    font-family: 'Big Shoulders';
    color: white;
    margin-left:10%;
    padding:20px;
    /* margin-right:10%; */
}
a {
    color: #FFFFFF;
    text-decoration: none;
}