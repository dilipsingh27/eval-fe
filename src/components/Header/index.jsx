import React from 'react';
import './Header.css';
import { Link } from 'react-router-dom';

const Header = () => {
    

    return (
        <div className='header main-padding'>
            <div className="logo">
                <Link to='/allEvents'>
                    <p className='para'>EVENTIFY</p>
                </Link>
            </div>
        </div>
    );
}; 

export default Header;