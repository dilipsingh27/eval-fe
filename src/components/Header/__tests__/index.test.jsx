import { render,screen } from '@testing-library/react';
import Header from '..'
import { BrowserRouter } from 'react-router-dom';

describe('Header', () => { 
    it('should render without crashing', () => {
        render(<BrowserRouter><Header/></BrowserRouter> );
        const appElement = screen.getByText('EVENTIFY',{exact:false});
        expect(appElement).toBeInTheDocument();
    });
});

describe('Header Snapshot', () => {
    it('should render forming a snapshot', () => {
        const { asFragment } = render(<BrowserRouter><Header/></BrowserRouter>);
        expect(asFragment()).toMatchSnapshot(); 
    });
});