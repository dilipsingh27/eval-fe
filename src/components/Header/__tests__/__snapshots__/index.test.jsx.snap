// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Snapshot should render forming a snapshot 1`] = `
<DocumentFragment>
  <div
    class="header main-padding"
  >
    <div
      class="logo"
    >
      <a
        href="/allEvents"
      >
        <p
          class="para"
        >
          EVENTIFY
        </p>
      </a>
    </div>
  </div>
</DocumentFragment>
`;
